### PR TITLE
LL-6318 - Ethereum Send flow step Summary: Add margin between GasPrice and GasLimit sections

### PR DESCRIPTION
**Pull request by jelbaz-ledger**
Fixes margin issues when a header is present at the top of the page (like when enabling experimental features). 

<details>
  <summary>Screenshot(s) 📸 </summary>
 <br/><br/>

*With "experimental features" header*

Simulator Screen Shot - iPhone 8 - 2021-07-09 at 08 57 09

-----

*Without any header*

Simulator Screen Shot - iPhone 8 - 2021-07-09 at 08 58 42

</details>

### Type

Bug Fix

### Context

[LL-6318]

### Parts of the app affected / Test plan

- Enable at least one experimental feature 
- Start the send ethereum crypto flow
- Go to the summary step
- Click on `Customize fees`
- Check the margin

[LL-6318]: https://ledgerhq.atlassian.net/browse/LL-6318

### CODE REVIEW

**Review by LFBarreto — APPROVED** [review]

File: src/families/ethereum/EditFeeUnitEthereum.js [hunk 1]
```diff
@@ -111,9 +111,7 @@ export default function EditFeeUnitEthereum({
 }
 
 const styles = StyleSheet.create({
-  root: {
-    flex: 1,
-  },
+  root: {},
```
Comment (juan-cortes) [inline src/families/ethereum/EditFeeUnitEthereum.js:114]: If we don't need any styles, drop this 🤷🏼

File: src/families/ethereum/EthereumCustomFees.js [hunk 2]
```diff
@@ -72,7 +72,6 @@ export default function EthereumCustomFees({ navigation, route }: Props) {
     parentAccount,
     route.params,
     transaction,
-    gasLimit,
```
Comment (juan-cortes) [inline src/families/ethereum/EthereumCustomFees.js:75]: We reaaally wanted to add `gasLimit` as a dependency here huh :trollface:

**Review by juan-cortes — DISMISSED** [review]

File: src/families/ethereum/EditFeeUnitEthereum.js (hunk 1 above)
Comment (jelbaz-ledger) [reply src/families/ethereum/EditFeeUnitEthereum.js]: done 👌

**Review by LFBarreto — APPROVED** [review]
:+1:

**Review by juan-cortes — APPROVED** [review]

**Review by ychen-ledger — APPROVED** [review]
Validated with :

- Android: HuaWei P30 - Android version: 10-smallest width: 320dp, 360dp(default)
- IOS: Iphone 6s Plus -- version 14.6
- IOS: emulator iphone 11/ Iphone 8 -- version 14.5
- Version 2.30.0

**Pull request merged**